lib/nostrum/struct/snowflake.ex

```diff
@@ -19,6 +19,8 @@ defmodule Nostrum.Struct.Snowflake do
   """
   @type t :: 0..0xFFFFFFFFFFFFFFFF
 
+  @discord_epoch 1_420_070_400_000
```

Comment:
I feel like we should maybe move this to the constants module, just in case we need to use this anywhere else, or if users want to use it for some reason.

Reply:
Fixed in 267c68b.